pulsar-client-cpp/lib/c/c_ConsumerConfiguration.cc

```diff
@@ -40,6 +40,19 @@ pulsar_consumer_type pulsar_consumer_configuration_get_consumer_type(
     return (pulsar_consumer_type)consumer_configuration->consumerConfiguration.getConsumerType();
 }
 
+void pulsar_consumer_configuration_set_schema_type(pulsar_consumer_configuration_t *consumer_configuration,
```

Comment:
@wolfstudy 

it seems that we never pass in properties for schema. that means that the properties users specify in go is not passed to c client and sent to broker.

also isn't it clear to have a method in consumer_configuration to set_schema_info and get_schema_info?